assignment_1/src/State.java

```diff
@@ -67,9 +67,68 @@ public String toString(){
 	public boolean isLeaf() {
 		return true;
 	}
-
+    /*
+        Check the values of the current position of the given agent and the values of the positions around the agent and returns the possible legal moves.
+    */
 	public Vector<String> legalMoves(int agent) {
```

Comment:
As stated in the assignment, could you also implement a function with signature as described below that gives the legalmoves back for the player who's turn it currently is?
![image](https://user-images.githubusercontent.com/34271434/192162056-33ea21ce-73f9-41a6-9211-0430839648fa.png)